<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>0</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="addAShape">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>250</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Add a Figure</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="size">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>261</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listOfShapes">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>181</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Square</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Circle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Triangle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>N-gon</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="showFigureInfoButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>430</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Show History of Shapes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="drawAllFiguresButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>370</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Draw all shapes from history</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="rotation">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>261</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>1</y>
        <width>501</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clearCanvas">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>310</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear canvas</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QPushButton" name="saveFiguresButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>144</y>
        <width>761</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Save Shapes to CSV</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadFiguresButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>761</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Shapes From CSV</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDisplay_Geometry_3">
    <property name="title">
     <string>Display Geometry :3</string>
    </property>
   </widget>
   <addaction name="menuDisplay_Geometry_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
